### ISSUE TITLE
Update fails with *.pkg file when using AppCenter. 

### ISSUE THREAD

**Issue by nagasunilt** [sparkle-project/Sparkle#1606]
### Description of the problem
I'm currently using Sparkle 2.x branch as our Mac app uses sandbox. I have .pkg installer and I use Visual Studio App Center to distribute the app. AppCenter allows you to send release info along with DSA signature instead of updating app cast file directly and AppCenter will automatically generate the xml file for Sparkle. app cast xml file looks like this for my app https://pastebin.com/CFnmHHeQ

Whenever there is an update, app prompts for it and downloads the new pkg file. But when Sparkle tries to install the downloaded file, I hit below error and update does not happen. It seems that Sparkle code is expecting `installationType` to be present in app cast xml file which does not exist in app cast xml generated by App Center. The check occurs here https://github.com/sparkle-project/Sparkle/blob/2.x/Autoupdate/SUInstaller.m#L132

I looked through the latest code in Sparkle 1.x and don't see the check for `installationType` so wondering when the rebase from 1.x happens for 2.x branch. I tried different ways for App Center to pick up `sparkle:installationType=package` but looks like it completes ignores this field since latest Sparkle 1.x does not require this field.

Instructions from App Center on Sparkle updates: 
https://docs.microsoft.com/en-us/appcenter/distribution/sparkleupdates

### Do you use Sandboxing in your app?
Yes

### Version of `Sparkle.framework` in the latest version of your app
2.x

### Version of `Sparkle.framework` in the old version of app that your users have (or N/A)
N/A

### Sparkle's output from Console.app
```
Error: Failed to create installer instance with error: Error Domain=SUSparkleErrorDomain Code=4005 "Found guided package installer but 'sparkle:installationType=package' was probably missing in the appcast item enclosure" UserInfo={NSLocalizedDescription=Found guided package installer but 'sparkle:installationType=package' was probably missing in the appcast item enclosure}
```

### Steps to reproduce the behavior
See description

**Comment by kornelski** [comment]
I didn't know Microsoft supported Sparkle, that's cool!

If you can't add the required attribute to `<enclosure>`, then the only solution would be to change `Sparkle.framework` not to require it.

It's unfortunate that this attribute is required in the first place. This is internal attribute used for secure communication with an XPC service leaking into user interface. It would be nice if Sparkle could be made to work without it.

**Comment by nagasunilt** [comment]
I made a PR to not check for installation type and match with 1.x implementation.

https://github.com/sparkle-project/Sparkle/pull/1607

**Comment by zorgiepoo** [comment]
@nagasunilt this will be 'eventually' resolved with non-archived pkg support (#1745,  https://github.com/sparkle-project/sparkle-project.github.io/pull/65)

**Comment by zorgiepoo** [comment]
Needs otherwise support ticket to Microsoft otherwise not actionable here.
